docs/cloud/webhooks.md

```diff
@@ -0,0 +1,277 @@
+
+Your goal when defining your event template is to produce a valid JSON object that
+defines (at minimum) the `event` name and the `resource["prefect.resource.id"]`, which
+are required of all events.  The simplest template is one in which these are statically
+defined.
+
+### Static webhook events
+
+Let's explore an example for static webhook templates.  Pretend that you are configuring
+a webhook that will notify Prefect when your `recommendations` model has been refreshed,
+so you can then send a Slack notification to your team and run a few subsequent
+deployments.  Those models are produced on a daily schedule by another team who are
+using `cron` for scheduling, they aren't able to use Prefect for their flows (yet!), but
+they are happy to add a `curl` to the end of their daily script to notify you.
+Because this webhook will only be used for a single event from a single resource, your
+template can be entirely static:
+
+```JSON
+{
+    "event": "model.refreshed",
+    "resource": {
+        "prefect.resource.id": "product.models.recommendations",
+        "prefect.resource.name": "Recommendations [Products]",
+        "producing-team": "Data Science"
+    }
+}
+```
+
+!!! tip "Make sure to produce valid JSON"
+
+    The output of your template, when rendered, should be a valid string that can be
+    parsed, for example, with `json.loads`.
+
+A webhook with this template may be invoked via _any_ of the HTTP methods, including a
+`GET` request with no body, so the team you are integrating with can include this at the
+end of their daily script:
+
+```bash
+curl https://api.prefect.cloud/hooks/AERylZ_uewzpDx-8fcweHQ
+```
+
+Each time the script "pings" the webhook like this, the webhook will produce a single
+Prefect event with that name and resource in your workspace.
+
+### Event fields that Prefect Cloud populates for you
+
+You may notice that you only had to provide the `event` and `resource` definition, which
+is not a completely fleshed out event.  Prefect Cloud will set default values for any
+missing fields, like `occurred` and `id`, so that you don't need to set them in your
+template.  Additionally, Prefect Cloud will always add the webhook itself as a related
+resource on all of the events it produces.
+
+If your template does not produce a `payload` field, the `payload` will default to a
+standard set of debugging information, including the HTTP method, headers, and body.
+
+### Dynamic webhook events
+
+Now let's say that after a few days you and the Data Science team are getting a lot of
+value from the automations you have set up with the static webhook.  You've agreed to
+upgrade this webhook to handle all of the various models that they produce.  It's time
+to add some dynamic information to your webhook template.
+
+Your colleages in Data Science have adjusted their daily `cron` scripts to `POST` a
+small body that includes the ID and name of the model that was just refreshed:
+
+```bash
```

Comment:
annoyingly this renders as a single line with a horizonatal scroll, can you make this multiline